packages/lib-utils/src/utils/common-fetch.ts

```diff
@@ -0,0 +1,90 @@
+import { CustomError, applyDefaults, applyOverrides } from '@monorepo/common';
+import { getUtilsConfig } from '../config';
+import type { CommonFetch, CommonFetchText, CommonFetchJSON } from '../types/fetch';
+
+class TimeoutError extends CustomError {
+  public constructor(url: string, ms: number) {
+    super(`Call to ${url} timed out after ${ms}ms`);
+  }
+}
+
+const defaultTimeout = 60_000; // one minute
```

Comment:
```suggestion
const defaultTimeout = 60_000;
```
60,000 is well known to be 60 seconds === 1 minute. I don't think we need these kind of comments imo.